pagopa/io-app
# chore: [IOAPPFD0-140] Remove `PictogramSection` from the codebase

## Short description
This PR removes any trace of `PictogramSection` from the codebase. The `PictogramSection` was used to render certain pictograms in the first level header, but it was only used in the `Messages` main section.

## List of changes proposed in this pull request
- Delete `PictogramSection` and its related files (SVG source files, React components, etc…)
- Update related READMEs
- Refactor `ScreenHeader` to accept generic `Pictogram` component
- Add new `Messages' pictogram while waiting for it to be removed once the header refactor is complete.

## How to test
Go to the **Profile → Design System → Pictograms**. You shouldn't see a `PictogramSection` block.
